subsys/bluetooth/host/classic/avdtp.c

```diff
@@ -970,24 +1100,24 @@ int bt_avdtp_l2cap_recv(struct bt_l2cap_chan *chan, struct net_buf *buf)
 /*A2DP Layer interface */
 int bt_avdtp_connect(struct bt_conn *conn, struct bt_avdtp *session)
 {
-	static const struct bt_l2cap_chan_ops ops = {
-		.connected = bt_avdtp_l2cap_connected,
-		.disconnected = bt_avdtp_l2cap_disconnected,
-		.encrypt_change = bt_avdtp_l2cap_encrypt_changed,
-		.recv = bt_avdtp_l2cap_recv
-	};
+	static const struct bt_l2cap_chan_ops ops = {.connected = bt_avdtp_l2cap_connected,
+						     .disconnected = bt_avdtp_l2cap_disconnected,
+						     .encrypt_change =
+							     bt_avdtp_l2cap_encrypt_changed,
+						     .recv = bt_avdtp_l2cap_recv};
```

Comment:
I think this is cleaner (hopefully clang-format will not fight against it):
```c
	static const struct bt_l2cap_chan_ops ops = {
		.connected = bt_avdtp_l2cap_connected,
		...
	};
```

Reply:
Unfortunately clang-format fight against it. I think clang-format do this way because it doesn't exceed the max line length. there are your style of codes in avdtp.c too, but clang-format doesn't change them.

Reply:
Is it really failing CI, or you're just getting a change suggestion from clang-format?

Reply:
I use the clang-format to format it locally, it changes as my codes. I am not sure whether the CI will fail, let me have a try later when I push new commit.

Reply:
Because there are two places to use the same `ops`, so I move it out of function scope to file scope. and I use your style, CI doesn't report errors.